bin/export_ot_baseline_metaanalysis.sh

```diff
@@ -4,12 +4,12 @@ scriptDir=$(cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 
 [ ! -z ${ATLASPROD_PATH+x} ] || ( echo "Env var ATLASPROD_PATH path to the directory needs to be defined." && exit 1 )
 [ ! -z ${BASELINE_META_DESTINATION+x} ] || ( echo "Env var BASELINE_META_DESTINATION path to the baseline baseline meta analysis directory needs to be defined." && exit 1 )
-[ ! -z ${BLUEPRINT_STUDIES+x} ] || ( echo "Env var BLUEPRINT_STUDIES comma separated blueprint baseline studies needs to be defined." && exit 1 )
-[ ! -z ${GTEX_STUDIES+x} ] || ( echo "Env var GTEX_STUDIES comma separated gtex baseline studies needs to be defined." && exit 1 )
+[ ! -z ${CELLTYPE_STUDIES+x} ] || ( echo "Env var BLUEPRINT_STUDIES comma separated blueprint baseline studies needs to be defined." && exit 1 )
```

Comment:
Please make this and following echo consistent (shouldn't be GTEX or BLUEPRINTS any more).

Reply:
Done